### PR TITLE
Improve node labels renderers

### PR DESCRIPTION
**Pull request by mdamien**
bench

* Improve correctness of node hover (we don't want the background to be smaller than the label)
* Make the label rendering ~70% faster (with the `min` metric which is a good stable indicator)

### CODE REVIEW

**Comment by sheymann** [comment]
@mdamien what does the min and med numbers mean?

**Comment by sheymann** [comment]
If it is the rendering time in ms, then min(current) > min(previous), so label rendering is slower or I'm missing something?

**Comment by sheymann** [comment]
If it is the rendering time in ms, then min(current) > min(previous), so label rendering is slower or I'm missing something?

**Comment by mdamien** [comment]
The time is in ms

* med: median of the sampled times
* min: mininum
* diff: difference with the minimum time

And you're right, it's slower, I missed something

**Comment by mdamien** [comment]
Uploaded the bench: https://rawgit.com/Linkurious/linkurious.js/benchmarks/bench/index.html

To use: open your dev console and then go the page

Code is here: https://github.com/Linkurious/linkurious.js/tree/benchmarks/bench

**Comment by sheymann** [comment]
Ok now I get 
min(current) = 43ms
min(previous) = 48ms

thus ~12% perf improvement

File: plugins/sigma.renderers.linkurious/canvas/sigma.canvas.hovers.def.js [hunk 1]
```diff
@@ -149,7 +153,7 @@
 
     function getTextWidth(text) {
       return settings('approximateLabelWidth') ?
-        0.5 * text.length * fontSize :
+        0.6 * text.length * fontSize :
```
Comment (mdamien) [inline plugins/sigma.renderers.linkurious/canvas/sigma.canvas.hovers.def.js]: Be larger for the background

**Comment by sheymann** [comment]
ok

**Pull request merged**